File: MockingPeople/Models/Person.cs
using System.Collections.Generic;

namespace MockingPeopleLibrary.Models
{
    public class Person
    {
        public int Id { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public List<Address> Addresses { get; set; }
        public List<ContactDevice> ContactDevices { get; set; }
        public override string ToString() => $"{FirstName} {LastName}";

        public Person()
        {
            
        }

        public Person(int id)
        {
            Id = id;
            Addresses = new List<Address>();
            ContactDevices = new List<ContactDevice>();
        }
    }
}

File: BindingNavigatorVBWinForms/Models/Person.vb
Imports System.Collections.Generic

Namespace Models
	Public Class Person
		Public Property Id() As Integer
		Public Property FirstName() As String
		Public Property LastName() As String
		Public Property Addresses() As List(Of Address)
		Public Property ContactDevices() As List(Of ContactDevice)
		Public Overrides Function ToString() As String
			Return $"{FirstName} {LastName}"
		End Function

		Public Sub New()

		End Sub

		Public Sub New(ByVal id As Integer)
			Me.Id = id
			Addresses = New List(Of Address)()
			ContactDevices = New List(Of ContactDevice)()
		End Sub
	End Class
End Namespace